contrib/exporters/secadvisor/mod/transform.go

```diff
@@ -38,7 +38,12 @@ func NewTransform(cfg *viper.Viper) (interface{}, error) {
 		return nil, err
 	}
 
-	resolver, err = NewResolveMulti(resolver.(Resolver))
+	resolver2, err := NewResolveVm(cfg)
+	if err != nil {
+		return nil, err
+	}
+
+	resolver, err = NewResolveMulti(resolver.(Resolver), resolver2.(Resolver))
```

Comment:
now that you have several resolvers then maybe use names such as:
- resolverRunc
- resolverVM (or resolverXen)

etc.

Reply:
OK